src/includes/performance/distributed-tracing/dotnet.mdx

```diff
@@ -0,0 +1,45 @@
+## Distributed Tracing
+
+Traces can bridge across multiple software services. Each span in a trace can be represented as a `sentry-trace` header, containing the trace id, span id, and sampling details. This `sentry-trace` header can be passed to downstream services so that they can create spans that are a continuation of the trace started in the originating service.
+
+To obtain a trace header from the span, use `ISpan.GetTraceHeader()` method, then pass it to the downstream service. If the communication happens over HTTP, it is recommended that you set the value to the `sentry-trace` HTTP header.
+
+```csharp
+var transaction = SentrySdk.StartTransaction("name", "op");
```

Comment:
on other SDKs, we always add the import of the classes, so users can copy/paste the whole code snippet.

https://develop.sentry.dev/sdk/philosophy/#assume-novice-developers